packages/cli/src/admin-sdk.ts

```diff
@@ -0,0 +1,224 @@
+import { createClient } from './autogen/client/createClient';
+import fse from 'fs-extra';
+import { kebabCase, omit } from 'lodash';
+import chalk from 'chalk';
+import { readAsJson, getFiles, getDirectories, replaceField } from './utils';
+import cliProgress from 'cli-progress';
+import { createHash } from 'crypto';
+import traverse from 'traverse';
+
+const multibar = new cliProgress.MultiBar(
```

Comment:
How about making these module level symbols like `UPPER_CASE`? This can help a lot when reading code, like it's easier to see what are local vars/arguments and what is global (which should be used with care).
```suggestion
const MULTIBAR = new cliProgress.MultiBar(
```